examples/src/tools/google_calendar.ts

```diff
@@ -31,22 +33,32 @@ export async function run() {
     new GoogleCalendarViewTool(googleCalendarParams),
   ];
 
-  const calendarAgent = await initializeAgentExecutorWithOptions(tools, model, {
-    agentType: "zero-shot-react-description",
-    verbose: true,
+  const llm = new ChatOpenAI({
+    temperature: 0,
+    apiKey: process.env.OPENAI_API_KEY,
+  });
+  const prompt: ChatPromptTemplate = await pull("hwchase17/openai-tools-agent");
+  const calendarAgent = await createOpenAIToolsAgent({
```

Comment:
Prefer `createReactAgent` from `@langchain/langgraph` here:

https://langchain-ai.github.io/langgraphjs/reference/functions/langgraph_prebuilt.createReactAgent.html

Reply:
done

Reply:
Would still prefer to use it and add install instructions as it's the latest and what people should be using long term. Can make modifications myself too.

Reply:
sure. I know what you mean to do. persuasive.
done.